src/foam/core/reflow/DAOPrompt.js

```diff
@@ -370,6 +429,30 @@ visible      },
         defaultCols;
         this.columnStorage[this.dao.of.id] = JSON.stringify(cols);
     },
+    {
+      name: 'updateDateCollectionWhere',
+      code: function() {
+        this.where = this.where
+          .replaceAll(this.workingDateWhere, '')
+          .replace(/\s{2,}/g, ' ').trim();
+        this.workingDateWhere = '';
+        if ( ( this.startDate || this.endDate ) && this.dateCollectionValue ) {
+          var propSplit = this.dateCollectionValue.split(',');
+          for ( let prop of propSplit ) {
+            if ( ! this.dao?.of?.getAxiomByName(prop) ) continue;
+            let bottomQ = this.startDate ? `${prop}>=${this.formatLocalYYYYMMDDHHMM(this.startDate)}` : '';
+            let topQ = this.endDate ? `${prop}<=${this.formatLocalYYYYMMDDHHMM(this.endDate)}` : '';
```

Comment:
`${prop}:${this.startDate?.toLocaleString('sv-SE').slice(0,10)}..${this.endDate?.toLocaleString('sv-SE').slice(0,10)}`